client/extensions/woocommerce/woocommerce-services/views/migrator-settings/index.js

```diff
@@ -0,0 +1,93 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { localize } from 'i18n-calypso';
+import { Button } from '@wordpress/components';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import ExtendedHeader from 'woocommerce/components/extended-header';
+import FeatureAnnouncement from 'components/migration/feature-announcement';
+import './style.scss';
+
+class MigratorSettingsRootView extends Component {
+	state = {
+		isMigrationNoticeVisible: false,
+		isMigrationNoticeDismissed: false,
+	};
+
+	componentWillMount() {
+		const migrationCookie = window.wpCookies.get( 'wcst-wcshipping-migration-dismissed' );
+		this.setState({ isMigrationNoticeDismissed: migrationCookie && parseInt( migrationCookie ) });
+	}
+
+	componentWillUnmount() {
+		this.props.restorePristineSettings(this.props.siteId);
+	}
+
+	addPackage = () => {
+		this.props.onChange();
+		this.props.addPackage(siteId);
+	};
+
+	showMigrationNotice = () => {
+		window.wpCookies.remove( 'wcst-wcshipping-migration-dismissed' );
+		this.setState({ isMigrationNoticeVisible: true });
+	}
+
+	onFeatureAnnouncementClosed = () => {
+		this.setState({ isMigrationNoticeVisible: false });
+	}
+
+	render() {
+		const {
+			translate,
+		} = this.props;
+
+		if ( ! this.state.isMigrationNoticeDismissed ) {
+			return null
+		}
+
+		return (
+			<div className="woocommerce-migrator-settings-root-view">
+				<ExtendedHeader
+					label={translate('Migration to WooCommerce Shipping')}
+					description={translate(
+						'You can now migrate to WooCommerce Shipping. This will provide you with a dedicated WooCommerce Shipping extension, which will carry over all your settings and shipping labels when you update.'
+					)}
+				>
+					<Button
+						className={classNames('button')}
+						onClick={ this.showMigrationNotice }
```

Comment:
A nice addition here could be adding a tracks event, not a train smash without it, but in case we want to see where people are initiating migrations from.

Reply:
done! [`71c9052` (#2796)](https://github.com/Automattic/woocommerce-services/pull/2796/commits/71c9052ec41822c962c97ceff23229ba4105e73c)